<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MenuPlanner+</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="central_layout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="landing_page">
       <layout class="QVBoxLayout" name="landingLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item>
         <spacer name="topSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="headerLabel">
          <property name="font">
           <font>
            <family>Poppins Medium</family>
            <pointsize>36</pointsize>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="text">
           <string>MenuPlanner+</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="descLabel">
          <property name="font">
           <font>
            <family>Centaur</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cook What You Crave, Love What You Eat.</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="imageLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>200</width>
            <height>200</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>C:/Users/LENOVO/Downloads/menu.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QPushButton" name="startButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>121</width>
            <height>41</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Poppins</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="bottomSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="main_page">
       <layout class="QVBoxLayout" name="mainPageLayout">
        <item>
         <widget class="QTabWidget" name="tabWidget">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="inputTab">
           <attribute name="title">
            <string>Plan Your Meal</string>
           </attribute>
           <layout class="QVBoxLayout" name="inputTabLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="formHeaderLabel">
              <property name="font">
               <font>
                <family>Poppins</family>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>üçΩÔ∏è Plan Your Meal</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QGridLayout" name="formGridLayout">
              <property name="spacing">
               <number>10</number>
              </property>
              <item row="0" column="0">
               <widget class="QLabel" name="nameLabel">
                <property name="text">
                 <string>üßëüèª Nama:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="nameInput">
                <property name="placeholderText">
                 <string>Masukkan nama Anda</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="timeLabel">
                <property name="text">
                 <string>üïí Waktu Makan:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <layout class="QHBoxLayout" name="timeRadioLayout">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="timeBreakfast">
                  <property name="text">
                   <string>Sarapan</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="timeLunch">
                  <property name="text">
                   <string>Makan Siang</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="timeDinner">
                  <property name="text">
                   <string>Makan Malam</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="timeSnack">
                  <property name="text">
                   <string>Snack</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="tasteLabel">
                <property name="text">
                 <string>üå∂Ô∏è Selera Rasa:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="tasteCombo">
                <item>
                 <property name="text">
                  <string>Manis</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Pedas</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Asin</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Asam</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="goalLabel">
                <property name="text">
                 <string>üéØ Tujuan Konsumsi:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <layout class="QHBoxLayout" name="goalRadioLayout">
                <property name="spacing">
                 <number>10</number>
                </property>
                <item>
                 <widget class="QRadioButton" name="goalBudget">
                  <property name="text">
                   <string>Hemat</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="goalFancy">
                  <property name="text">
                   <string>Mewah</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="goalHealthy">
                  <property name="text">
                   <string>Diet</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="prepTimeLabel">
                <property name="text">
                 <string>‚è±Ô∏è Waktu Persiapan:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QComboBox" name="prepTimeCombo">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>150</width>
                  <height>30</height>
                 </size>
                </property>
                <item>
                 <property name="text">
                  <string>Cepat</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Sedang</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Lama</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="buttonLayout">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QPushButton" name="generateButton">
                <property name="text">
                 <string>Generate Menu</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="surpriseButton">
                <property name="text">
                 <string>Surprise Me!</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="resetButton">
                <property name="text">
                 <string>Reset</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="resultLabel">
              <property name="font">
               <font>
                <family>Poppins Medium</family>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="text">
               <string>üßæ Hasil Rekomendasi:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextEdit" name="resultTextEdit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
              <property name="placeholderText">
               <string>Rekomendasi menu akan muncul di sini...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="historyTab">
           <attribute name="title">
            <string>Meal History</string>
           </attribute>
           <layout class="QVBoxLayout" name="historyTabLayout">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>20</number>
            </property>
            <property name="topMargin">
             <number>20</number>
            </property>
            <property name="rightMargin">
             <number>20</number>
            </property>
            <property name="bottomMargin">
             <number>20</number>
            </property>
            <item>
             <widget class="QLabel" name="historyHeaderLabel">
              <property name="font">
               <font>
                <family>Poppins</family>
                <pointsize>20</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>üìú Meal History</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableWidget" name="historyTable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="historyButtonLayout">
              <property name="spacing">
               <number>10</number>
              </property>
              <item>
               <widget class="QPushButton" name="exportCsvButton">
                <property name="text">
                 <string>Export to CSV</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="exportPdfButton">
                <property name="text">
                 <string>Export to PDF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clearHistoryButton">
                <property name="text">
                 <string>Clear History</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExportCsv"/>
    <addaction name="actionExportPdf"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionClearHistory"/>
    <addaction name="actionClearForm"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionUserGuide"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <family>Inter</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <action name="actionExportCsv">
   <property name="text">
    <string>Export to CSV</string>
   </property>
  </action>
  <action name="actionExportPdf">
   <property name="text">
    <string>Export to PDF</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionClearHistory">
   <property name="text">
    <string>Clear History</string>
   </property>
  </action>
  <action name="actionClearForm">
   <property name="text">
    <string>Reset Form</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionUserGuide">
   <property name="text">
    <string>User Guide</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>